README.md

```
@@ -427,6 +429,12 @@ Instructions available [here](https://github.com/tsparticles/preact/#readme)
 
 ### ReactJS
 
+#### `@tsparticles/nextjs`
+
+[![npm](https://img.shields.io/npm/v/@tsparticles/nextjs?style=for-the-badge)](https://www.npmjs.com/package/@tsparticles/nextjs) [![npm](https://img.shields.io/npm/dm/@tsparticles/nextjs?style=for-the-badge)](https://www.npmjs.com/package/@tsparticles/nextjs)
+
+Instructions available [here](https://github.com/tsparticles/tsparticles/tree/main/wrappers/nextjs#readme)
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)